cmd/cluster-init/update_jobs.go

```diff
@@ -0,0 +1,70 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"path/filepath"
+
+	prowconfig "k8s.io/test-infra/prow/config"
+	"sigs.k8s.io/yaml"
+
+	"github.com/openshift/ci-tools/pkg/jobconfig"
+)
+
+type InfraPeriodics struct {
+	Periodics []prowconfig.Periodic `json:"periodics,omitempty"`
+}
+
+func updateJobs(o options) error {
+	var config prowconfig.JobConfig
+	updatePresubmits(o, &config)
+	updatePostsubmits(o, &config)
+	updateInfraPeriodics(o, &config)
+
+	metadata := RepoMetadata()
+	return jobconfig.WriteToDir(filepath.Join(o.releaseRepo, "ci-operator", "jobs"), metadata.Org, metadata.Repo, &config)
+}
+
+func updatePresubmits(o options, config *prowconfig.JobConfig) {
+	config.PresubmitsStatic = map[string][]prowconfig.Presubmit{
+		"openshift/release": {generatePresubmit(o.clusterName)},
+	}
+}
+
+func updatePostsubmits(o options, config *prowconfig.JobConfig) {
+	config.PostsubmitsStatic = map[string][]prowconfig.Postsubmit{
+		"openshift/release": {generatePostsubmit(o.clusterName)},
+	}
+}
+
+func updateInfraPeriodics(o options, config *prowconfig.JobConfig) {
+	config.Periodics = []prowconfig.Periodic{generatePeriodic(o.clusterName)}
+}
+
+func periodicExistsFor(o options) (bool, error) {
```

Comment:
Why do we need this?

Reply:
(A counterpoint: move the current jobs from infra periodics to the correct `openshift/release` periodic file, then we do not have to write or maintain custom code)

Reply:
True, this won't even work properly when jobs are in the correct file. I'll just get rid of this and only check the cluster's directory for duplicate validation. I am going to also allow the option to pass in an `update` flag which will require the cluster to already exist. That will come in a future PR.